tests/pytorch/nightly/dlrm.libsonnet

```diff
@@ -82,94 +88,122 @@ local utils = import 'templates/utils.libsonnet';
+        %s 
+      ||| % utils.toCommandString(self.paramsOverride.trainCommand),
     ),
   },
-  local convergence_common = |||
-    --raw-data-file=/datasets/criteo-kaggle-mm/train.txt \
-    --processed-data-file=/datasets/criteo-kaggle-mm/kaggleAdDisplayChallenge_processed.npz \
-    --memory-map \
-    --arch-sparse-feature-size=16 \
-    --arch-mlp-bot="13-512-256-64-16" \
-    --arch-mlp-top="512-256-1" \
-    --data-generation=dataset \
-    --data-set=kaggle \
-    --loss-function=bce \
-    --round-targets=True \
-    --learning-rate=0.1 \
-    --mini-batch-size=128 \
-    --print-freq=1024 \
-    --print-time \
-    --test-mini-batch-size=16384 \
-    --test-freq=101376 \
-    --use-tpu \
-    --num-indices-per-lookup=1 \
-    --num-indices-per-lookup-fixed \
-    --tpu-model-parallel-group-len 8 \
-    --tpu-metrics-debug \
-    --tpu-cores=8 |& tee dlrm_logs.txt
-    acc=`grep Testing dlrm_logs.txt | tail -1 | grep -oP 'best \K[+-]?([0-9]*[.])?[0-9]+'`
-    echo 'Accuracy is' $acc
-    test $(echo $acc'>'78.75 | bc -l) -eq 1  # assert cls acc higher than 78.75
-  |||,
+
   local criteo_kaggle = common.Convergence {
+    paramsOverride+:: {
+      tpuCores: 8,
+      miniBatchSize: 128,
+      tpuModelParallelGroupLen: 8,
+      ArchSparseFeatureSize: 16,
+      ArchMlpBot: '13-512-256-64-16',
+      ArchAlpTop: '512-256-1',
+      NumIndicesPerLookup: 1,
+      trainCommand+: [
+        '--raw-data-file=/datasets/criteo-kaggle-mm/train.txt',
+        '--processed-data-file=/datasets/criteo-kaggle-mm/kaggleAdDisplayChallenge_processed.npz',
+        '--memory-map',
+        '--data-generation=dataset',
+        '--print-time',
+        '--test-mini-batch-size=16384',
+        '--test-freq=101376',
+        '--data-set=kaggle',
+        '--loss-function=bce',
+        '--round-targets=True',
+        '--learning-rate=0.1',
+        '--print-freq=1024',
+        '--no-save',
+        '--max-epoch=1',
+      ],
+    },
     command: utils.scriptCommand(
       |||
+        set +e
         apt-get install -y bc
         pip install onnx
         git clone --recursive https://github.com/pytorch-tpu/examples.git
-        python examples/deps/dlrm/dlrm_tpu_runner.py \
-          %(convergence_common)s
-      ||| % convergence_common
+        %s 
+        --raw-data-file=/datasets/criteo-kaggle-mm/train.txt \
+        --processed-data-file=/datasets/criteo-kaggle-mm/kaggleAdDisplayChallenge_processed.npz \
+        --memory-map \
+        |& tee dlrm_logs.txt
+        acc=`grep Testing dlrm_logs.txt | tail -1 | grep -oP 'best \K[+-]?([0-9]*[.])?[0-9]+'`
+        echo 'Accuracy is' $acc
+        test $(echo $acc'>'78.75 | bc -l) -eq 1  # assert cls acc higher than 78.75
+      ||| % [
+        utils.toCommandString(self.paramsOverride.trainCommand),
+      ]
     ),
   },
   local v3_8 = {
     accelerator: tpus.v3_8,
   },
   configs: [
-    dlrm + v3_8 + one_core + timeouts.Hours(3) + mixins.Experimental,
```

Comment:
Can you mark these experimental per #598?

Reply:
done!